calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/good_sample.jpg" alt="good_sample" width = 700px height = 300px></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/bad_sample.jpg" alt="bad_sample" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Good initial calibration.</p></td>
+    <td><p style="text-align: center;">Bad initial calibration.</p></td>
+  </tr>
+</table>
+
+### Delete previous lidar-radar pair
+
+During the calibration, if there are any mismatched pairs (e.g., a human appearing in front of both the radar and lidar), the user can click the `Delete previous lidar-radar pair` button to remove the previous outlier pair.
+
+Using the tutorial rosbag as an example, we can delete the latest radar and lidar pair by clicking this button. The before and after changes should look like the images shown below.
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/delete1.jpg" alt="delete1" width = 700px ></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/delete2.jpg" alt="delete2" width = 700px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Before deletion.</p></td>
+    <td><p style="text-align: center;">After deletion.</p></td>
+  </tr>
+</table>
+
+The console should also show the following text.
+
+```text
+[marker_radar_lidar_calibrator]: The last track was successfully deleted. Remaining converged tracks: 4
+```
+
+### Metric plotter
+
+The tool also provides a metric plotter for real-time visualization shown below.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/metric_plotter1.jpg" alt="metric_plotter1" width="500">
+</p>
+
+The subplots at the top display the cross-validation errors, while the bottom subplot shows the average errors in the calibration procedure. Plotting for the average errors begins after three pairs have been collected. For the cross-validation errors, plotting starts after four pairs have been collected.
+
+Consider the left-top subplot, which plots the cross-validation errors for distances between radar estimation and lidar estimation, as an example of how these errors are calculated. When the x-axis value is 3, it indicates that we estimate the transformation using 3 samples from the 5 converged tracks. We then calculate the distance errors with all of the converged tracks. This process is repeated for 5 choose 3 (5C3) times, which totals 10 times, and the errors are then averaged. The light blue area represents the standard deviation of the 10 calculated distance errors.
+
+### Send calibration
+
+The user can click the `Send calibration` button once it is enabled. However, it is recommended to stop the calibration when the line in the cross-validation error is converged. Therefore, in this tutorial, we run the calibration process until the bag is finished. Once the calibration is ended, the console should show a similar result as below.
```

Comment:
What converges it not a line (lines do nor converge). It is either the curve or the name of the metric

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cfbbf6514c6d5d9802a9aa014c455d08e9e5c335